app/Homplexity.hs

```diff
@@ -33,12 +34,35 @@ import HFlags
 
 import Paths_homplexity (version)
 import Data.Version (showVersion)
+import qualified Prelude as P
+import Prelude hiding (head, id, div)
+#ifdef HTML_OUTPUT
```

Comment:
Elegant way to limit dependencies.
It would be better to move HTML-related code to a different module though.